sevsnpmeasure/cli.py

```diff
@@ -94,9 +98,12 @@ def main() -> int:
 
     try:
         sev_mode = SevMode.from_str(args.mode)
-        ld = guest.calc_launch_digest(sev_mode, args.vcpus, vcpu_sig, args.ovmf,
-                                      args.kernel, args.initrd, args.append, args.snp_ovmf_hash,
-                                      vmm_type=vmm_type)
+
+        if args.dump_vmsa is True and sev_mode not in [SevMode.SEV_SNP, SevMode.SEV_SNP_SVSM]:
+            parser.error("--dump-vmsa in only available in SNP mode")
```

Comment:
Maybe allow dumping VMSA in all modes except SEV? (SEV-ES and all SNP variants should use VMSAs)

Reply:
Is this feature useful at all? I was just thinking to remove it from the PR.
I just added it for debugging.
It can be extended to other modes as well, of course.

Reply:
Done